docs/index.html

```diff
@@ -29,7 +29,7 @@
      [<i class="fa fa-coffee"></i> Buy us a coffee](https://ko-fi.com/dennisdoomen){: .btn .btn--paypal}
      <br/>
      A very extensive set of extension methods that allow you to more naturally specify the expected outcome of a TDD or
-     BDD-style unit tests. Targets .NET Framework 4.7, as well as .NET 6, .NET Standard 2.0 and 2.1.'
+     BDD-style unit tests. Targets .NET Framework 4.7, as well as .NET 6, .NET 8, .NET Standard 2.0 and 2.1.'
```

Comment:
Should this be just a note that we are ".NET 8 ready", since we don't target .NET 8

Reply:
Yes, something like that. Or we should rephrase all of the references to the frameworks. Like "Works with .NET Standard 2.0 and higher, .NET Framework 4.7 and higher and .NET 6 and higher"

Reply:
I also think there are a couple of other places where we mention the version, including the NuGet description and such.

Reply:
You were right. I found three more places 😄